src/main/java/net/dries007/tfc/config/ServerConfig.java

```diff
@@ -359,6 +378,43 @@
 
         hotWaterHealAmount = builder.apply("hotWaterHealAmount").comment("An amount that sitting in hot water will restore health, approximately twice per second.").defineInRange("hotWaterHealAmount", 0.08, 0.0, 20.0);
 
+        innerBuilder.pop().push("saplings");
+
+        globalSaplingGrowthModifier = builder.apply("globalSaplingGrowthModifier").comment("Modifier applied to the growth time of every (non-fruit) sapling").defineInRange("globalSaplingGrowthModifier", 1d, 0d, Double.MAX_VALUE);
+        globalFruitSaplingGrowthModifier = builder.apply("globalFruitSaplingGrowthModifier").comment("Modifier applied to the growth time of every fruit tree sapling").defineInRange("globalFruitSaplingGrowthModifier", 1d, 0d, Double.MAX_VALUE);
+
+        saplingGrowthDays = new EnumMap<>(Wood.class);
+        for (Wood wood : Wood.VALUES)
+        {
+            final String valueName = String.format("%sSaplingGrowthDays", wood.getSerializedName());
+            saplingGrowthDays.put(wood, builder.apply(valueName).comment(String.format("Days for a %s tree sapling to be eligible to grow", wood.getSerializedName())).defineInRange(valueName, wood.defaultDaysToGrow(), 0, Integer.MAX_VALUE));
+        }
+        fruitSaplingGrowthDays = new EnumMap<>(FruitBlocks.Tree.class);
+        for (FruitBlocks.Tree tree : FruitBlocks.Tree.values())
+        {
+            final String valueName = String.format("%sSaplingGrowthDays", tree.getSerializedName());
+            fruitSaplingGrowthDays.put(tree, builder.apply(valueName).comment(String.format("Days for a %s tree sapling to be eligible to grow", tree.getSerializedName())).defineInRange(valueName, tree.defaultDaysToGrow(), 0, Integer.MAX_VALUE));
+        }
+        bananaSaplingGrowthDays = builder.apply("bananaSaplingGrowthDays").comment("Days for a banana tree sapling to be eligible to grow").defineInRange("bananaSaplingGrowthDays", 6, 0, Integer.MAX_VALUE);
```

Comment:
"Eligible to grow" is a strange turn of phrase and does not explain what it means. Is it growth time? Average growth time? Does it need to be eligible multiple times before a tree grows?

Also why are these all measured in days (`growthDays`); where below the crop times (`growthTime`) are measured in ticks and all other devices are measured in ticks (`ticks`). Use the existing standard (`fooTicks`)

Reply:
eligible to grow = it is done counting ticks and needs a random tick to grow

Reply:
Not possible to change it from days now without ugly breaking changes to RegistryWood that would leave the constructor ambiguous as the type would still be int and supplier<int>